docs/model_quality.mdx

```diff
@@ -0,0 +1,131 @@
+To effectively understand how accurate an emissions "measurement" is, given the many layers of assumptions and allocations involved in the calculation, is critical both for assessing provider quality and as a gating factor to incorporate data into corporate and regulatory reporting.
+
+## Model Quality Components
+
+Each modeled request should include:
+- Grid mix model quality (1-5)
+- Organization model quality (1-5) 
+- Property model quality (1-5)
+- Ad stack model quality (1-5)
+- Average ad platform / vendor quality (1-5)
+- Input granularity score (1-5)
+
+## Grid mix
+
+Grid intensity can fluctuate significantly on an hourly basis due to the variable nature of both renewable energy sources (sun, wind) and electricity demand. To achieve effective decarbonization, having hourly data is critical. However, this isn't broadly available in many countries.
+
+| Data source and timescale  | Model Quality  |
+| -------------------------- | -------------- |
+| Worldwide                  | 1              |
+| Country, monthly or annual | 2              |
+| Country, daily             | 3              |
+| Country + region, daily    | 4              |
+| Country + region, hourly   | 5              |
+
+## Organization
+
+Criteria for accurate sustainability data at the organization level:
+
+* An organization should provide a sustainability report that details its full carbon footprint, including scopes 1, 2, and all scope 3 categories. This report should clearly detail methodology for each category.
+
+* Any adjustments to the calculations for RECs, PPAs, offsets, or carbon credits should detail exactly what was purchased, from whom, and on what timeframe. Location-based scope 2 data should always be provided.
+
+* A sustainability report should be published within 6 months of the end of the previous calendar year to be considered for sustainability purposes.
+
+| Report element                  | Model Quality |
+| ------------------------------- | ------------- |
+| Scope 1 and 2, market-based     | +1 |
+| Location-based scope 2          | +1 |
+| All scope 3 categories provided | +1 |
+| Lines of business broken out    | +1 |
+
+## Vendor / Ad Platform Quality
+
+An ad platform should provide 1) requests received, 2) requests sent (traffic shaping), and 3) emissions data from datacenters or cloud providers
```

Comment:
You out this in another section, and I think its good to be explicit. Add `The vendor / ad platform score starts at 1 and increments based on modeled components.`